daemon/daemon.go

```diff
@@ -101,6 +101,24 @@ func NewDaemon(cfg *config.Config) *Daemon {
 		return nil
 	}
 
+	// get containerd version to make sure containerd has started
+	_, err = ctrdClient.Version(context.Background())
+	if err != nil {
+		logrus.Errorf("failed to get containerd version: %v", err)
+		return nil
+	}
+
+	if cfg.Snapshotter != "" {
+		ctrd.SetSnapshotterName(cfg.Snapshotter)
+	}
+
+	if err = checkSnapshotter(ctrd.CurrentSnapshotterName(), filepath.Join(cfg.HomeDir, "containerd/root")); err != nil {
+		logrus.Errorf("check snapshotter driver falied: %v", err)
```

Comment:
failed to check snapshotter driver: %v ?

Reply:
Have done it!